rollup.config.js

```diff
@@ -0,0 +1,34 @@
+import copy from 'rollup-plugin-copy';
+import del from 'rollup-plugin-delete';
+import dynamicImportVars from '@rollup/plugin-dynamic-import-vars';
+import glob from 'glob-all';
+import resolve from '@rollup/plugin-node-resolve';
+
+const jsGlob = [
+	'@(components|controllers|directives|helpers|mixins|templates)/**/*.js',
+	'!**/*@(test|axe|visual-diff).js',
+];
+const nonJsGlob = [
+	'@(components|controllers|directives|helpers|mixins|templates)/**/*.*',
+	'*.*',
+	'!**/*.@(js|md|json)',
+	'!**/screenshots/**/*',
+];
+
+export default {
+	input: glob.sync(jsGlob),
+	output: { dir: 'build', format: 'es', preserveModules: true },
```

Comment:
`preserveModules` keeps the generated files looking very similar to their originals, and keeps all `node_modules` imported files in their original folder.

The [docs](https://rollupjs.org/guide/en/#outputpreservemodules) recommend not doing this, and explicitly naming all inputs ("entry points") - but I found that by doing **both**, we get the best of both worlds 😄.